debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -371,7 +373,7 @@ private boolean schemaChanged(List<PgProto.DatumMessage> messageList, Table tabl
         // go through the list of columns from the message to figure out if any of them are new or have changed their type based
         // on what we have in the table metadata....
         return messageList.stream().filter(message -> {
-            String columnName = message.getColumnName();
+            final String columnName = PostgresTableIdTransformer.INSTANCE.fromSqlQuoted(message.getColumnName());
```

Comment:
My update to DBZ-297 includes the suggestion made here: https://github.com/debezium/debezium/pull/260#discussion_r127855155 - I don't believe the transformer should be stored in the Column.  In essence, the column instance contains the correct information (the column name in the correct case) - it is only that when sending this string to Postgres we have to escape it so it is a quoted identifier.  Handy reference here: https://www.postgresql.org/docs/9.0/static/sql-syntax-lexical.html

Note, it isn't just capital letters that need quoting - we also need to ensure that if someone creates a table or column name which is an SQL keyword it is quoted too.  Therefore it made sense to me to quote everything when its sent to Postgres.